tests/cpp.test

```diff
@@ -307,4 +307,4 @@
 33078 multi_line_8.cfg                 cpp/multi_line.cpp
 33079 multi_line_9.cfg                 cpp/multi_line.cpp
 33080 multi_line_10.cfg                cpp/multi_line.cpp
-33081 bug_i_552.cfg                    cpp/bug_i_552.cpp
+33082 empty.cfg                        cpp/gh555.cpp
```

Comment:
right... a common problem of the test suite. Then we have to rename the output file to fix the mismatch.